app/src/main/res/layout/fragment_panel.xml

```diff
@@ -1,12 +1,7 @@
 <RelativeLayout
   xmlns:android="http://schemas.android.com/apk/res/android"
   xmlns:app="http://schemas.android.com/apk/res-auto"
-  xmlns:tools="http://schemas.android.com/tools"
-  android:layout_width="wrap_content"
-  android:layout_margin="@dimen/bpkSpacingBase"
-  android:layout_gravity="center_vertical"
-  android:layout_height="wrap_content"
-  tools:ignore="HardcodedText">
```

Comment:
lets not remove the `tools:ignore:"HardcodedText"` rule

Reply:
I've added it again where it was necessary @amolgupta